usr/share/rear/lib/uefi-functions.sh

```diff
@@ -46,7 +46,7 @@ function build_bootx86_efi {
         Log "Did not find grub-mkimage (cannot build bootx86.efi)"
         return
     fi
-    if [ -f /etc/slackware-version ] ; then
+    if [[ -f /etc/slackware-version ]] || [[ -f /etc/fedora-release ]] ; then
         # slackware grub doesn't have linuxefi module
```

Comment:
Please enhance the comment that also Fedora 29 does not have linuxefi module
plus a URL to the matching issue https://github.com/rear/rear/issues/1996
so that others who read the code later better unerstand the reason behind.